<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpcUaClientDialog</class>
 <widget class="QDialog" name="OpcUaClientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OPC UA Client Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="4">
      <widget class="QPushButton" name="connectPushButton">
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Port:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Server:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="disconnectPushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Disconnect</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="6">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>OPC UA Nodes</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QTreeWidget" name="opcUaNodesTreeWidget">
          <attribute name="headerDefaultSectionSize">
           <number>200</number>
          </attribute>
          <column>
           <property name="text">
            <string>Node</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Value</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date time stamp</string>
           </property>
           <property name="textAlignment">
            <set>AlignHCenter|AlignVCenter|AlignCenter</set>
           </property>
          </column>
          <item>
           <property name="text">
            <string notr="true">RBS.BeamRangeShifter.01</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <item>
            <property name="text">
             <string notr="true">State</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ValueCode</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">ValueThickness</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
           <item>
            <property name="text">
             <string notr="true">HeartBeat</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </item>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="opcUaServerPathLineEdit">
       <property name="text">
        <string notr="true">opc.tcp://localhost</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QSpinBox" name="opcUaServerPortSpinBox">
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>4840</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="6">
      <widget class="QCheckBox" name="opcUaConnectAtStartupCheckBox">
       <property name="text">
        <string>Connect at StartUp</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
